packages/hint-no-html-only-headers/README.md

```diff
@@ -21,6 +21,9 @@ HTTP headers:
 * `X-WebKit-CSP`
 * `X-XSS-Protection`
 
+In case of a JavaScript file, `Content-Security-Policy` and
+`X-Content-Security-Policy` will be ignored.
```

Comment:
Small addition to explain why these are ignored. 
```suggestion
`X-Content-Security-Policy` will be ignored since CSP is also
relevant to workers.
```